R/test-server.R

```diff
@@ -49,14 +48,13 @@ isModuleServer <- function(x) {
 #'   stopifnot(myreactive() == 4)
 #'   stopifnot(output$txt == "I am 4")
 #'   # Any additional arguments, below, are passed along to the module.
-#' }, multiplier = 2)
+#' })
 #' @export
-testServer <- function(app = NULL, expr, ...) {
+testServer <- function(app = NULL, expr, args = rlang::list2()) {
```

Comment:
Why not just `list()` here? I think `rlang::list2()` will confuse people.

Reply:
Sounds good. Changing to `list()`